src/components/Home/Aditya/Week4/pages/Week1/components/CreateTaskCard/createTaskCardStyles.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { grey, primary, secondary, tertiary, warning, white } from '../../colorExports';
+
+export const NewTask = styled.div`
```

Comment:
it is common practice to name styled components as Wrappers you can name as StyledWrapper or component name + Wrapper